libs/dal/src/repositories/job/job.repository.ts

```diff
@@ -34,7 +34,7 @@ export class JobRepository extends BaseRepository<JobDBModel, JobEntity, Enforce
       stored.push(this.mapEntity(created));
     }
 
-    await this.MongooseModel.insertMany(stored, { ordered: true });
+    await this.insertMany(stored, true);
```

Comment:
I wonder how will we handle if there will be an exception on the bulk error, for example, if we have an array of 100 inserts and the first one causes an error. 
At the moment with an unordered insert, the other ones will be executed in parallel while in an ordered insert the execution will stop.

[Reference](https://www.mongodb.com/docs/manual/reference/method/db.collection.insertMany/#error-handling)

Reply:
I believe that is the behaviour we want, no? Why do you feel that would be a problem? 

[A previous discussion ](https://github.com/novuhq/novu/pull/3329#discussion_r1184748745)

Reply:
Sorry actually, I thought I was commenting executionDetailsRepository.insertMany.
I will continue the comment there :)